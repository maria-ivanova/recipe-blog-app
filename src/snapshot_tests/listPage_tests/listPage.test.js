import React from 'react';
import renderer from 'react-test-renderer';
import Routing from '../helpers/router.js';
import ListPage from '../../components/listPage.js';

jest.mock('../../components/itemsList.js', () => 'ItemsList');

describe('List page Component', () => {
    it('should render page layout', () => {
        const tree = renderer.create(
            <Routing>
                <ListPage match={{ params: { categoryName: 'test category' } }}/>
            </Routing>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    })

})
