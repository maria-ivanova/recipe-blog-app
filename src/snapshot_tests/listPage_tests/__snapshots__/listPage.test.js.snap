// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List page Component should render page layout 1`] = `
Array [
  <section
    className="sec"
  >
    <div
      className="banner_item"
    >
      <h2
        className="title"
      >
        test category
      </h2>
    </div>
  </section>,
  <ItemsList
    filerCriterion="category"
    hiddenPageTitle="yes"
    secondFilterCriterion="test category"
  />,
]
`;
